src/Network.cpp

```diff
@@ -891,11 +891,28 @@ Network::Netresult Network::get_scored_moves(
     if (ensemble == DIRECT) {
         assert(symmetry >= 0 && symmetry <= 7);
         result = get_scored_moves_internal(planes, symmetry);
+    } else if (ensemble == MULTI_AVG) {
+        for (int r = 0; r < 8; ++r) {
+            Netresult tmpresult = get_scored_moves_internal(planes, r);
+            result.winrate += tmpresult.winrate;
+            result.policy_pass += tmpresult.policy_pass;
+
+            for (auto idx = size_t{0}; idx < BOARD_SQUARES; idx++) {
+                result.policy[idx] += tmpresult.policy[idx];
+            }
+        }
+
+        for (auto idx = size_t{0}; idx < BOARD_SQUARES; idx++) {
+            result.policy[idx] /= 8;
+        }
+
+        result.winrate /= 8;
+        result.policy_pass /= 8;
     } else {
-        assert(ensemble == RANDOM_SYMMETRY);
+        assert(ensemble == RANDOM_ROTATION);
```

Comment:
This looks like a merge conflict fail. This branch probably should be left unchanged